third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -340,7 +340,7 @@ func TestAccContainerCluster_withMasterAuthorizedNetworksConfig(t *testing.T) {
 		CheckDestroy: testAccCheckContainerClusterDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccContainerCluster_withMasterAuthorizedNetworksConfig(clusterName, []string{}, "cidr_blocks = []"),
```

Comment:
I think this whole param can be removed now

Reply:
ehh probably. I want to minimise the changes here (+ get this in before the EOD), so I'm gonna leave it.